pixel-point/gatsby-tailwind-starter
# Gatsby plugin image + deps update

# Describe the changes

This PR updates deps (result of `npm outdated`) + replaces `gatsby-image` with `gatsby-plugin-image`.

`StaticImage` has been added to `index.jsx` for demonstration purposes.

# Step to test

1. Pull down the changes
2. Run `npm i`
3. Run `npm run start`
4. See that everything works 🎉 

# Refs

Closes task in [Notion](https://www.notion.so/pixelpoint/Use-gatsby-plugin-image-in-our-starter-729d89b9bdb84ba48f9b101ec53a510b)
